database/seeders/DmsSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\DMS;
+
+class DmsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        //
+        $dms = [
+            ['sender_id' => 1, 'receiver_id' => 2, 'product_id' => 1, 'body' => 'Hello'],
+            ['sender_id' => 2, 'receiver_id' => 1, 'product_id' => 1, 'body' => 'Hi'],
```

Comment:
I suggest use factory in this one. If you want many records.

Reply:
I changed DmsSeeder for using factory with [a9ebde7](https://github.com/kredo-school/16th_rankshty/pull/72/commits/a9ebde7d5b4c5623f0e92411ba54d0e421eb5f93).
In addition that, I canged NotificationSeeder.php in order to make all patterns of dummy notifications.